.github/workflows/dev_container_feature_release.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Release dev container features & Generate Documentation"
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Publish Features"
+        uses: devcontainers/action@v1
+        with:
+          publish-features: "true"
+          base-path-to-features: "./dev-container-feature/src"
+          generate-docs: "true"
+          
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create PR for Documentation
+        id: push_image_info
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -e
+          echo "Start."
+          # Configure git and Push updates
+          git config --global user.email github-actions@github.com
+          git config --global user.name github-actions
```

Comment:
Shouldn't this be related to daprbot?
@artursouza @yaron2 @ItalyPaleAle  thoughts?

Reply:
This is ok with me, but I am also ok with changing to use Dapr bot. Unsure if these values are used as convention and there’s any benefit.

Reply:
If you have different values that you'd like used for this I'm happy to push an update - let me know 😄

Reply:
It is OK to be via github-actions.